core/math/math_funcs.h

```diff
@@ -263,6 +263,16 @@ class Math {
 		return value;
 	}
 
+	static _ALWAYS_INLINE_ double fmod_TAU(double x) {
+		static constexpr double tau_reversed = 1.0 / Math_TAU;
```

Comment:
```suggestion
		static constexpr double TAU_INV = 1.0 / Math_TAU;
```

Or something like that.